scripts/release.sh

```diff
@@ -206,12 +207,38 @@ if [ -n "$OVERRIDE_COMMIT_HASH" ]; then
   alias_tag=
 fi
 
+# This variable will be used to flag if the helm chart dependencies have been
+# been updated.
+_helm_dependencies_updated=false
 for name in $IMAGES; do
   image_basename=$($NIX_EVAL -f . images.$BUILD_TYPE.$name.imageName | xargs)
   image=$image_basename
   if [ -n "$REGISTRY" ]; then
     image="${REGISTRY}/${image}"
   fi
+
+  if [ "$_helm_dependencies_updated" = false ]; then
```

Comment:
would be tidier to move this part to a script which does only helm update and calling it from here.

Reply:
Keeping it in the scripts/release.sh for now.